woocommerce/woocommerce-gutenberg-products-block
# Change `shippingRatesLoading` to `isLoadingRates`

Since whether the shipping rates are loading is a boolean value, the name should be updated to reflect this. I have changed every instance of `shippingRatesLoading` to `isLoadingRates` - the context it is used in is always related to shipping so it's not going to be confusing.

This is a follow up to https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5896#discussion_r819531870 
### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Smoke test cart and checkout and ensure you can change shipping method.
2. Change your country and ensure the shipping method selector is disabled while the new rates load.
3. Try with express payment methods, ensure checking out works fine.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
